.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will build Waltz with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Java CI with Gradle
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
Can we use an LTS version instead of the latest? The idea behind that would be to avoid surprises in the latest version and rely on a version that has long-term support enabled.